app/scripts/modules/core/src/application/modal/PlatformHealthOverride.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import { isEqual } from 'lodash';
 
-import { IServerGroupCommand } from 'core/serverGroup';
 import { HelpField } from 'core/help/HelpField';
 
 export interface IPlatformHealthOverrideProps {
-  command: IServerGroupCommand;
+  command: IPlatformHealthOverrideCommand;
```

Comment:
Is this used all over the place?   Can we refactor this to take a `healthProviderNames: string[]` prop instead of taking a command object?

Reply:
Good idea. Looks like it's just in a couple places. I will update it to drop the command in favor of that field.